### ISSUE TITLE
WorkspaceId (guid) needs to be entered manually each time a PBIX is uploaded

### ISSUE THREAD

**Issue by hanssens** [Azure-Samples/power-bi-embedded-integrate-report-into-web-app#15]
In the ProvisionSample, case 6, there is a null check for `workspaceId`:

```
    case '6':
        // ... removed for brevity 

        if (string.IsNullOrWhiteSpace(workspaceId))
        {
            Console.Write("Workspace ID:");
            workspaceId = Console.ReadLine();
            Console.WriteLine();
        }
```

This is always null, because workspaceId is never read from the app config. Because copy/paste isn't possible in the console, and the workspaceId is a guid, this is quite tedious to manually have to type each time.

**Comment by wbreza** [comment]
You should be able to copy/paste from the console.  In the build and console you can right click in the console window header to get access to the clipboard.

Feel free to submit a pull request to add this check

**Comment by hanssens** [comment]
@wbreza thanks, see above - the PR is already there.

**Comment by wbreza** [comment]
Resolved with #14
